Sources/TuistCache/Cache/Cache.swift

```diff
@@ -37,9 +38,9 @@ public final class Cache: CacheStoraging {
         }
     }
 
-    public func fetch(hash: String) -> Single<AbsolutePath> {
+    public func fetch(hash: String, userConfig: Config) -> Single<AbsolutePath> {
```

Comment:
Nitpick: I'd remove the `user` prefix to be consistent with the rest of the codebase.

Reply:
updated.